styles.css

```diff
@@ -376,3 +388,198 @@ a {
   background-color: #2230d2;
   color: #fff;
 }
+
+/* desktop */
+.nested-nav-ul {
+  display: flex;
+  align-items: center;
+  padding-right: 60px;
+}
+
+.nested-nav-li {
+  margin-right: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 20px;
+}
+
+/* MOBILE */
+@media (max-width: 768px) {
+  .nested-nav-li {
+    display: none;
+  }
+
+  .nested-nav-ul {
+    padding-right: 10px;
+  }
+
+  .project-image-web {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
```

Comment:
- [ ] Kindly ensure to also use Grid to layout the elements on the page. Currently, you have only used flexbox.